### ISSUE TITLE
Training Yolo Algorithm

### ISSUE THREAD

**Issue by Abdelrahmanalii** [pjreddie/darknet#1906]
I was reading many blogs about Yolo Algorithm but I have a bit confusion about how we label the training data,, I will write my explanation and want to know if its right or not.

in yolo algorithm if we decide to have the output layer to be something like 19*19*n
so we must make a ground truth label vector for each of the 19*19 cell in the input image 
in order to be able to train the yolo algorithm ??
and I see many people just assign output vector with ground truth label of the desired object and regardless any other regions in the image so how would the algorithm be trained like that ?
